### PR TITLE
Fix/build

### PR DESCRIPTION
**Pull request by alex-glv**
- Added travis build script
- Created a directory layout to build / test / deploy the service
- Running tests from travis

### CODE REVIEW

File: test/unit.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from os.path import expanduser
+
+from util.Docker import Docker
+
+
+class GoServices(unittest.TestCase):
+    def test_go(self):
+        script_dir = os.path.dirname(os.path.realpath(__file__))
+        code_dir = script_dir + "/.."
+        home = expanduser("~")
```
Comment (idcrosby) [inline test/unit.py]: this isn't used... (realises this was copied from my code... feels bad)

File: test/util/Dredd.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from util.Docker import Docker
+
+
+class Dredd:
+    image = 'weaveworksdemos/openapi'
+    container_name = ''
+
+    def test_against_endpoint(self, json_spec, endpoint_container_name, api_endpoint,
+                              mysql_container_name):
```
Comment (idcrosby) [inline test/util/Dredd.py]: payment doesn't use mysql or any database at all (at this point)

File: test/util/Dredd.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        self.container_name = Docker().random_container_name('openapi')
+        command = ['docker', 'run',
+                   '-h', 'openapi',
+                   '--name', self.container_name,
+                   '--link', mysql_container_name,
```
Comment (idcrosby) [inline test/util/Dredd.py]: ibid

File: test/unit.py (hunk 1 above)
Comment (alex-glv) [inline test/unit.py]: It is used, in .travis.yml 
```
  - ./test/test.sh unit.py
```

**Comment by idcrosby** [comment]
LGTM

**Pull request merged**